docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -118,3 +118,73 @@ Then, replace the contents of `pages/index.js` with the code below, restart the
   </TabItem>
 </Tabs>
 
+### Connected Components
+
+<Tabs>
+  <TabItem title="Next.js 13.4+">
+
+To use Connected componnets in app directory, make sure you re-export `Authenticator` as a Client Component by placing "use client" at the top of the file and set up with `Amplify.configure`, for example:
+
+    <ExampleCode>
+
+    ```tsx
+    // Authenticator.jsx
+    'use client';
+
+    import { Amplify } from 'aws-amplify';
+
+    import { Authenticator } from '@aws-amplify/ui-react';
+    import '@aws-amplify/ui-react/styles.css';
+
+    import awsExports from '../src/aws-exports';
+    Amplify.configure(awsExports);
```

Comment:
`Amplify.configure` can be done serverside, right?

Reply:
It can. I think if people are using some Amplify SSR features, they need to run it on the server side as well. For us, we do not have anything specific on the server now.

Reply:
Since this docs is just for UI components, do you think it makes sense to remove `Amplify.configure` from here so that we don't make an opinion on where customer should do it?